src/applications/vaos/tests/e2e/appointment-list.cypress.spec.js

```diff
@@ -254,13 +254,14 @@ describe('VAOS appointment list refresh', () => {
       cy.axeCheckBestPractice();
     });
 
-    it.skip('should navigate to requested appointment details', () => {
+    it('should navigate to past appointment details', () => {
       cy.get('[data-cy=appointment-list-item]')
         .first()
         .findByText(/Details/i)
         .focus()
         .click();
-      cy.findByText(/Request detail/i).should('exist');
+      cy.findByText(/Appointment detail/i).should('exist');
```

Comment:
I believe previously the first item in the past appointments list was an EC request, but now those are gone and fallen off the list, so this is the accurate test now.